spark/src/main/java/org/opensearch/sql/spark/data/constants/SparkConstants.java

```diff
@@ -8,13 +8,22 @@
 public class SparkConstants {
```

Comment:
could add the change in https://github.com/opensearch-project/sql/pull/2216/files#diff-6eca1b0df87741e862bd1455b8a39e503489990984f8d790ebf7de7016986b86

Reply:
added